cli/spinner.ts

```diff
@@ -74,13 +74,35 @@ export interface SpinnerOptions {
  */
 export class Spinner {
   #spinner: string[];
-  /** The message to display next to the spinner. */
-  message: string;
   #interval: number;
   #color?: Color;
   #intervalId: number | undefined;
   #active = false;
 
+  /**
+   * The message to display next to the spinner.
+   * This can be changed while the spinner is active.
+   *
+   * @example
+   * ```ts
+   * import { Spinner } from "@std/cli/spinner";
+   *
+   * const spinner = new Spinner({ message: "Working..." });
+   * spinner.start();
+   *
+   * for (let step = 0; step < 5; step++) {
+   *   // do some work
+   *   await new Promise((resolve) => setTimeout(resolve, 1000));
+   *
+   *   spinner.message = `Finished Step #${step}`;
+   * }
+   *
+   * spinner.stop();
+   * spinner.message = "Done!";
```

Comment:
Just realized this should be `console.log` since setting the message after `stop` won't do anything. I'll fixl